tests/utils/browser.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const feedBackDocs = async (testName = 'allLogs') => {
+  const script = `(async () => {
+  const allDbList =  await indexedDB.databases();
+  const metaDbList = allDbList.filter(db => db.name.includes('pouch_medic-user') && db.name.endsWith('-meta'))
+  const feedBack = []
+  for (const idx in metaDbList){
+    const db = metaDbList[idx]
+    const nameStripped = db.name.replace('_pouch_','');
+    const metaDb = new PouchDB(nameStripped);
+    const docs = await metaDb.allDocs({ include_docs:true });
```

Comment:
This could be really massive, which isn't a big problem because it's all on the local machine, and it's for testing not for production usage, but I think we could get a decent performance gain when running tests by using the doc ID index that PouchDB natively has. eg:

```suggestion
    const docs = await metaDb.allDocs({ include_docs: true, startkey: 'feedback-', endkey: 'feedback-\ufff0' });
```

Firstly, this means you get fewer docs from the db, but also you don't need the filter below any more.